rogueutil.h

```diff
@@ -876,7 +877,16 @@ getUsername(void)
                 return ret;
         return NULL;
 #else /* _WIN32 */
-        return getlogin();
+#ifdef __linux__
```

Comment:
Shouldn't be needed, getpwuid() and getuid() seem to be available across *nixes.

Reply:
Do I use __unix__ instead of __linux__?

Reply:
Just leave it as an `#else` to `_WIN32`.